src/vendor/GLTFLoader.js

```diff
@@ -80,7 +80,7 @@ THREE.GLTFLoader = ( function () {
 
 		},
 
-		parse: function ( data, path, onLoad, onError ) {
+		parse: function ( url, data, path, onLoad, onError ) {
```

Comment:
I kept wanting to make this change but couldnt figure out a clean way to do it without altering the signature of parse but still get the url. I guess modifying it is fine but id add it as the last param to keep compatability. Would prefer to try and come up with a solution we can submit upstream though...

Reply:
Added the url as an instance param instead, to avoid changing the signature.

Most of the other changes are about MOZ_ALT_MATERIALS vs upstream https://gist.github.com/brianpeiris/59f9707f87c679e94e3f5411664b57d2